app/packages/components/src/components/ThemeProvider/index.tsx

```diff
@@ -120,18 +120,19 @@ let theme = extendJoyTheme({
 });
 
 export const useTheme = () => {
-  return theme.colorSchemes[useRecoilValue(fos.theme)].palette;
+  return theme.colorSchemes["light"].palette;
```

Comment:
Is this an intended change? This looks like it will break theme selection.

Reply:
Theme selection isn't working as intended since it's tightly coupled to recoil.  There was back and forth on how to successfully implement this, but I think the recoil integration should be a separate ticket.